examples/imdb_smt.py

```diff
@@ -0,0 +1,106 @@
+import torch
+import torch.nn as nn
+
+from ignite.metrics import Loss, Accuracy
+from sklearn.preprocessing import LabelEncoder
+
+from torch.optim import Adam
+from torch.utils.data import Dataset, DataLoader, ConcatDataset
+
+from torchnlp.datasets import imdb_dataset  # type: ignore
+from torchnlp.datasets import smt_dataset  # type: ignore
+
+from slp.data.collators import DACollator
+from slp.data.transforms import SpacyTokenizer, ToTokenIds, ToTensor
+from slp.modules.daclassifier import DAClassifier, DALoss
+from slp.modules.rnn import WordRNN
+from slp.trainer.trainer import DATrainer
+from slp.util.embeddings import EmbeddingsLoader
+
+
+class DatasetWrapper(Dataset):
+    def __init__(self, dataset, name):
+        self.dataset = dataset
+        self.name = 0 if name == 'imdb' else 1
+        self.label = 'sentiment' if name == 'imdb' else 'label'
+        self.transforms = []
+        self.label_encoder = (LabelEncoder()
+                              .fit([d[self.label] for d in dataset]))
+
+    def map(self, t):
```

Comment:
there is also LambdaDataset under slp/data/datasets.py so you don't need to implement map all the time